app/policies/roles/institution/health_organization.json.erb

```diff
@@ -10,20 +10,14 @@
             "institution:read",
             "institution:update",
             "institution:delete",
-            "institution:registerDevice",
-            "institution:createSite",
-            "institution:createRole"
+            "institution:createRole",
+            "institution:readUsers"
           ],
           "delegable": true
         },
         {
-          "resource": [
-            "site?institution=<%= institution_id %>",
-            "device?institution=<%= institution_id %>",
-            "testResult?institution=<%= institution_id %>",
-            "encounter?institution=<%= institution_id %>"
-          ],
-          "action": "*",
+          "resource": "testResult?institution=<%= institution_id %>",
+          "action": "testResult:medicalDashboard",
```

Comment:
The point is that health organizations does not generate test results, so in order to view any in the dashboard they need to ask for that permission to another institution.